src/modules/jumpbox/main.tf

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+resource "random_id" "jumpbox-keyvault" {
+  byte_length = 12
+}
+
+resource "azurerm_key_vault" "jumpbox-keyvault" {
+  name                       = format("%.24s", lower(replace("${var.keyvault_name}${random_id.jumpbox-keyvault.id}", "/[[:^alnum:]]/", "")))
```

Comment:
@brooke-hamilton now appending a random alphanumeric string to the passed in keyvault name